frontend/src/redux/museumArtworkActions.js

```diff
@@ -15,6 +15,62 @@ const convertToArtworkInfo = (artwork) => {
   return artworkInfo;
 };
 
+function filterArtworks(artworks) {
+  const artworksArr = [];
+  for (const artwork of artworks) {
+    if (artwork.thumbnail != null) {
+      artworksArr.push(convertToArtworkInfo(artwork));
+    }
+  }
+  return artworksArr;
+}
+
+// based off of https://github.com/kjschmidt913/AIC/blob/master/script.js
+function getQuery(limit) {
+  return {
+    'resources': 'artworks',
+    'fields': [
```

Comment:
What is the purpose of these fields? Are they the only ones retrieved?

Reply:
Yes this is specifying which fields the fetch call should return.  I figured it would be better to only get the info we need (and we can of course add or delete any of these fields)

Reply:
Could you add artist and description?

Reply:
Yes!

Reply:
thanks!